src/amo/components/AddonBadges/styles.scss

```diff
@@ -8,6 +8,11 @@
   @include respond-to(large) {
     grid-column: 2;
     grid-row: 1;
+
+    .Addon-theme & {
+      grid-column: auto;
```

Comment:
Thanks for this change. It makes more sense because of how theme previews need to be as wide as possible.